sdk/playwrighttesting/azure-resourcemanager-playwrighttesting/src/main/java/com/azure/resourcemanager/playwrighttesting/PlaywrightTestingManager.java

```diff
@@ -213,7 +217,7 @@ public PlaywrightTestingManager authenticate(TokenCredential credential, AzurePr
                 .append("-")
                 .append("com.azure.resourcemanager.playwrighttesting")
                 .append("/")
-                .append("1.0.0-beta.3");
+                .append("1.0.0-beta.1");
```

Comment:
This version seems not correct.
This is set by `package-version` in `FluentJavaSettings`. It's passed in in autorest, but not in TypeSpec.
Possible solutions:
1. Add `package-version` to emitter option
2. In codegen, determine package version(if `version` is absent)
3. in `updateVersion` script, bump version. Though it'll be a bit hard to decide the actual xxManager class..

Reply:
We can do an emitter option, though sending it across `tsp-client` may be tricker.
Or a hack via env var...

Reply:
env var should do the trick. Yeah, `version` seems no qualified candidate for an emitter option.

Reply:
Seems we have to strip the lib name for tsp-client:
--emitter-options="service-name=Playright" (without `@azure-tools/typespec-java`)

Reply:
Great. So we only need to add (maybe a hidden `package-version`), and set this in script.